Updating Readme to make writePoints function work

manorie opened the pull request:
Hello,

I have tried this example as it is for now. Encountered an error that AddPoint function expects only one argument. Checked projects GoDoc and updated the example in your error handling style. It works like this.

###### Required for all non-trivial PRs
- [X] Rebased/mergable
- [X] Tests pass
- [ ] CHANGELOG.md updated
(seemed unnecessary as it is related to the README example function fix)
- [X] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

###### Required only if applicable
_You can erase any checkboxes below this note if they are not applicable to your Pull Request._
- [ ] [InfluxQL Spec](https://github.com/influxdata/influxdb/blob/master/influxql/README.md) updated
- [ ] Provide example syntax
- [ ] Update man page when modifying a command
- [ ] [InfluxData Documentation](https://github.com/influxdata/docs.influxdata.com): issue filed or pull request submitted \<link to issue or pull request\>

e-dard commented on client/README.md:
```diff
@@ -142,12 +142,19 @@ func writePoints(clnt client.Client) {
+		)
+		
+		if err != nil {
+			log.Fatalln("Error: ", err)
```
This can just be `log.Fatal(err)`.

manorie replied:
Yes this bothers me also, I just made it to be consistent with previous error handlings in the same sniplet. Let me clear it.

e-dard replied:
@manorie you're welcome to clean the rest of them up too. 😄

manorie commented:
As I have seen you are open to improvements @e-dard I changed log functions, added small improvements to make it more readable and clean (imo). Hope you like it.

e-dard commented on client/README.md:
```diff
@@ -119,16 +119,21 @@ NOTE: You can specify a RetentionPolicy as part of the batch points. If not
+	}
 
 	for i := 0; i < sampleSize; i++ {
+		rand.Seed(uint64(i))
```
What is this line for?

manorie replied:
It is a good catch. I wanted to give a true randomness here by incrementing the seed with iterator. Every time this function runs, will create a new graph. But the mistake I made here is to get confused about how to do it, I think taking the seed with the way of the last commit will change the sequence in every run.

e-dard commented:
This looks great @manorie. Thanks for your work here!

manorie commented:
You are welcome @e-dard . I will check open issues to see if I can contribute more in my spare time. Hope I can contribute more.
